bundles/core-commands/commands/get.js

```diff
@@ -6,6 +6,7 @@ module.exports = (srcPath) => {
   const ItemType = require(srcPath + 'ItemType');
 
   return {
+    usage: 'get item [container]',
```

Comment:
This may need to be `get <item> [container]` if `<>` means required and `[]` means optional.